chapters/annotations.tex

```diff
@@ -798,7 +794,7 @@ \subsection{Component Instance}\label{component-instance}
 
 \subsection{Extends clause}\label{extends-clause}
 
-Each extends-clause (and short-class-definition, as stated in \cref{annotations-for-graphical-objects}) may have layer specific annotations which describe the rendering of the base class' icon and diagram layers in the derived class.
+Each extends-clause (and short class definition, as stated in \cref{annotations-for-graphical-objects}) may have layer specific annotations which describe the rendering of the base class' icon and diagram layers in the derived class.
```

Comment:
I believe that both extends-clause and short-class-definition should refer to the grammar constructs. Having one grammar construct and one grammatical construct seems odd.

Reply:
(By the way, the change I had in mind, but didn't have time to implement yet, was to turn the extends-clause into `extends`-clause, which would be a better match with _short class definition_.)